### PR TITLE
Revise language to consolidate sign up process

### PR DESCRIPTION
**Pull request by rlskoeser**

### CODE REVIEW

**Comment by maltevogl** [comment]
Text looks good to me. Question was if the mailing list sign-up is automatic? Also as far as I know membership to the organization is not automatic either?

**Comment by rlskoeser** [comment]
No, not automatic — the last time we discussed it we decided it was low enough volume to keep doing manually, and @jdamerow said she likes to see the names of people who are joining. We did discuss making sure more than one person can handle processing new sign ups so it isn't all relying on a single person, but IDK if we actually did anything about that.

**Comment by jdamerow** [comment]
I started adding people to the mailing list when they sign up (I also added a note to the Google form noting as much). @maltevogl also has access to the Google sheet with registrations so he can (and does) add people as well.

File: join.md [hunk 1]
```diff
@@ -6,11 +6,6 @@ title: Join DHTech
...
 
-<ul>
-	<li><p>You can participate in our workhops by following the respective invitation links, no registration necessary.</p></li>
-	<li><p>To join our Slack channel and our <a href="http://github.com/dh-tech">GitHub organization</a> please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSeAe1MlCPOCVlY7YsJUcLv1_tDBtEOBNJ2vhCe6aWcvAl-ehQ/viewform">this form</a>.</p></li>
-	<li><p>To subscribe to the low-volume announcements mailing list, go to the <a href="https://listserv.gwdg.de/mailman/listinfo/dh-tech-announce">DH-Tech Announce mailing list interface.</a></p></li>
-</ul>
-
-
+* You can participate in our workhops by following the event invitation links; no registration is necessary.
+* Fill out the [DHTech signup form](https://docs.google.com/forms/d/e/1FAIpQLSeAe1MlCPOCVlY7YsJUcLv1_tDBtEOBNJ2vhCe6aWcvAl-ehQ/viewform) to be added to our Slack, [GitHub organization](http://github.com/dh-tech), and be subscribed to our low-volume [announcements mailing list](https://listserv.gwdg.de/mailman/listinfo/dh-tech-announce).
```
Comment (jdamerow) [inline join.md:10]:
```suggestion
* Fill out the [DHTech signup form](https://docs.google.com/forms/d/e/1FAIpQLSeAe1MlCPOCVlY7YsJUcLv1_tDBtEOBNJ2vhCe6aWcvAl-ehQ/viewform) to be added to our Slack and be subscribed to our low-volume [announcements mailing list](https://listserv.gwdg.de/mailman/listinfo/dh-tech-announce).
```
We stopped adding people to GitHub so we should probably take that out here (we should discuss the GitHub process though, who gets access etc.).

**Pull request merged**